src/Raven.Server/ServerWide/ClusterCommandsVersionManager.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using Raven.Server.ServerWide.Commands;
+using Raven.Server.ServerWide.Commands.ConnectionStrings;
+using Raven.Server.ServerWide.Commands.ETL;
+using Raven.Server.ServerWide.Commands.Indexes;
+using Raven.Server.ServerWide.Commands.Monitoring.Snmp;
+using Raven.Server.ServerWide.Commands.PeriodicBackup;
+using Raven.Server.ServerWide.Commands.Subscriptions;
+
+namespace Raven.Server.ServerWide
+{
+    public static class ClusterCommandsVersionManager
+    {
+        public const int DefaultVersion = 400;
+
+        public static readonly int MyCommandsVersion;
+
+        public static int Max(IEnumerable<int> values)
```

Comment:
What does this gives over `Enumerable.Max`?

Reply:
Oops..  I've searched the Max function in Math.